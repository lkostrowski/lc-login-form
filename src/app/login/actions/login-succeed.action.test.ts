import { createLoginSucceedAction } from './login-succeed.action';

describe('Login Succeed Action', function() {
    it('Should match snapshot', () => {
        expect(
            createLoginSucceedAction({
                email: 'some-email@test.com',
                uid: 'user-id',
            }),
        ).toMatchSnapshot();
    });
});
