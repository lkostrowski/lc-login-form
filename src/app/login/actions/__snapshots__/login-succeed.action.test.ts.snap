// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Succeed Action Should match snapshot 1`] = `
Object {
  "payload": Object {
    "email": "some-email@test.com",
    "uid": "user-id",
  },
  "type": "login:login-succeed",
}
`;
